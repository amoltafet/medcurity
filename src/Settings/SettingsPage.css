.settingsMenuContainer {
    margin-left: 2vmax;
    margin-right: 2vmax;
    margin-bottom: 2vmax;
    margin-top: 2vmax;
}