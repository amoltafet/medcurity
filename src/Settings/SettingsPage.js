import {Container} from 'react-bootstrap'
import React from 'react';
import './SettingsPage.css';
import 'bootstrap/dist/css/bootstrap.min.css'
import MenuBar from '../MenuBar';
import SettingsMenu from './SettingsMenu';

/**
* Creates and displays each the main settings page. 
* @return {SettingsPage}
*/
const SettingsPage = () => {
    return (
        <>
        <MenuBar></MenuBar>
        <Container fluid="md" className="settingsMenuContainer">
        <SettingsMenu></SettingsMenu>
        </Container> 
        </>
    );
}
export default SettingsPage;