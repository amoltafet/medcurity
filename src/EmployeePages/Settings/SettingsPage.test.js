import renderer from 'react-test-renderer';
import SettingsPage from './SettingsPage';
import React from 'react';

test('renders SettingsPage', () => {
  const tree = renderer.create(<SettingsPage />).toJSON();
  expect(tree).toMatchSnapshot();
});