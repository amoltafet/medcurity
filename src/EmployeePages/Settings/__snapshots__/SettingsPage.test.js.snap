// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SettingsPage 1`] = `
Array [
  <div
    className="card-group"
  >
    <div
      className="logo card"
    >
      <a
        href="/dash"
      >
        <img
          alt=""
          className="card-img-top MedcurityLogo"
          src="/Medcurity_Logo.png"
        />
      </a>
    </div>
    <div
      className="pillz card"
    >
      <div
        className="justify-content-end nav card-header-pills nav-pills"
        onKeyDown={[Function]}
      >
        <div
          className="navPills uvs-left uvs-right nav-item"
        >
          <a
            className="menubarFont nav-link"
            data-rb-event-key="/settings"
            href="/settings"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Settings
          </a>
        </div>
        ,
        <div
          className="navPills uvs-left uvs-right nav-item"
        >
          <a
            className="menubarFont nav-link"
            data-rb-event-key="/"
            href="/"
            onClick={[Function]}
            onKeyDown={[Function]}
          >
            Logout
          </a>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="settingsMenuContainer container-md"
  >
    <div
      className="settingsRow row"
    >
      <div
        className="shadowTab col-sm-2"
      >
        <div
          className="flex-column marginTop  nav nav-pills"
          onKeyDown={[Function]}
          role="tablist"
        >
          <div
            className="selectedSetting nav-item"
          >
            <a
              aria-controls="left-tabs-example-tabpane-first"
              aria-selected={true}
              className="nav-link active"
              data-rb-event-key="first"
              href="#"
              id="left-tabs-example-tab-first"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="tab"
            >
              <img
                alt=""
                className="profileImage  rounded-circle"
                src="/user.png"
                variant="top"
              />
            </a>
          </div>
          <div
            className="unselectedSetting nav-item"
          >
            <a
              aria-controls="left-tabs-example-tabpane-second"
              aria-selected={false}
              className="nav-link"
              data-rb-event-key="second"
              href="#"
              id="left-tabs-example-tab-second"
              onClick={[Function]}
              onKeyDown={[Function]}
              role="tab"
            >
              Tab 2
            </a>
          </div>
        </div>
      </div>
      <div
        className="dropShadow col-sm-8"
      >
        <div
          className="settingsMenuContainer container"
        >
          <div
            className="tab-content"
          >
            <div
              aria-hidden={false}
              aria-labelledby="left-tabs-example-tab-first"
              className="fade tab-pane active show"
              id="left-tabs-example-tabpane-first"
              role="tabpanel"
            >
              <form
                className=""
              >
                <div
                  className="usernameInput form-group"
                >
                  <small
                    className="usernameText form-text"
                  >
                    UserName
                  </small>
                  <input
                    className="form-control"
                    id="exampleForm.ControlTextarea1"
                    placeholder="username"
                  />
                </div>
                <div
                  className="emailInput form-group"
                >
                  <small
                    className="emailText form-text"
                  >
                    Email
                  </small>
                  <input
                    className="form-control"
                    id="exampleForm.ControlTextarea1"
                    placeholder="email"
                  />
                </div>
              </form>
            </div>
            <div
              aria-hidden={true}
              aria-labelledby="left-tabs-example-tab-second"
              className="fade tab-pane"
              id="left-tabs-example-tabpane-second"
              role="tabpanel"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
]
`;
